cmd/metadata/main.go

```diff
@@ -212,3 +224,33 @@ func createViews(ctx context.Context, database golibConfig.Database) ([]string,
 
 	return views, nil
 }
+
+func readCustomHasuraConfigs(ctx context.Context, database golibConfig.Database) ([]hasura.Request, error) {
```

Comment:
It looks like `readCustomHasuraConfig` should be exported function in `go-lib`. Something like that:

```go
func ReadCustomConfig(ctx context.Context, database golibConfig.Database, hasuraConfig string) (hasura.Request, error) {
 ....
}
// OR
func ReadCustomConfigs(ctx context.Context, database golibConfig.Database, hasuraConfigDir string) ([]hasura.Request, error) {
 ....
}
```

But it's not requirement. It's my thoughts.

Reply:
I was going by where and how sql views are handled and that seemed like the place to put it.

Can move the function to go-lib, no problem.